### DISCUSSION TITLE
OGG audio not playing on my website?

### DISCUSSION THREAD

**Discussion by NeoRootPain** [qutebrowser/qutebrowser#7715] [Q&A]
I'm running through this weird issue with audio on Qutebrowser

My site ( https://rootpain.com/changelog.html ) plays audio just fine on other browsers, or if I change the format to MP3 or WAV, but the moment I use Qutebrowser, audio does not play, except for a single page on my entire website

I don't think it got nothing to do with coding mistakes since the audio plays fine on other browsers and even on mobile

There's also this weird bug in which the live website audio works fine, but when I'm editing it directly from the desktop, it does not, but I'm not sure if this is qute's fault or Linux fault

Currently using Artix Linux.

**Comment by mschilli87** [comment]
I get music on both the Changelog and the Guestbook page.

**Reply by NeoRootPain** [reply]
That's weird
What might be causing this weird issue on my end?
